components/project-content.js

```diff
@@ -1,47 +1,28 @@
 import { RichText } from './rich-text';
+import { Banner } from './banner';
+
 export const ProjectContent = ({
-  title,
+  projects: { title, thumbnail, description, projectLink },
```

Comment:
Here we can add `technologies`and with `map`we can use `<Label>{technology}</Label>` in label container